src/structures/APIMessage.js

```diff
@@ -149,6 +165,9 @@ class APIMessage {
     if (this.isMessage) {
       // eslint-disable-next-line eqeqeq
       flags = this.options.flags != null ? new MessageFlags(this.options.flags).bitfield : this.target.flags.bitfield;
```

Comment:
```suggestion
      flags = this.options.flags !== null ? new MessageFlags(this.options.flags).bitfield : this.target.flags.bitfield;
```

Reply:
`!=` may be checking for null **and** undefined (check eslint: eqeqeq) which might be intended behaviour. I'm not sure.

Reply:
![image](https://user-images.githubusercontent.com/23212967/102712299-5b6dab00-42c0-11eb-8692-f6f8797cdf3f.png)
Indeed, it is checking for both null and undefined values

Reply:
Awesome.